config/default/pyproject.toml.j2

```diff
@@ -52,15 +52,34 @@ Zope = [
   'Products.SiteAccess', 'Shared', 'Testing', 'ZPublisher', 'ZTUtils',
   'Zope2', 'webdav', 'zmi',
   # Zope dependencies
-  'Acquisition', 'DateTime', 'transaction', 'zExceptions', 'ZODB', 'zope.component',
-  'zope.configuration', 'zope.container', 'zope.deferredimport', 'zope.event',
-  'zope.exceptions', 'zope.globalrequest', 'zope.i18n', 'zope.i18nmessageid',
-  'zope.interface', 'zope.lifecycleevent', 'zope.location', 'zope.publisher',
-  'zope.schema', 'zope.security', 'zope.site', 'zope.traversing', 'AccessControl',
+  'AccessControl', 'Acquisition', 'AuthEncoding', 'beautifulsoup4', 'BTrees',
+  'cffi', 'Chameleon', 'DateTime', 'DocumentTemplate', 'ExtensionClass',
+  'MultiMapping', 'multipart', 'PasteDeploy', 'Persistence', 'persistent',
+  'pycparser', 'python-gettext', 'pytz', 'RestrictedPython', 'roman', 'six',
+  'soupsieve', 'transaction', 'waitress', 'WebOb', 'WebTest', 'WSGIProxy2',
+  'z3c.pt', 'zc.lockfile', 'ZConfig', 'zExceptions', 'ZODB', 'zodbpickle',
+  'zope.annotation', 'zope.browser', 'zope.browsermenu', 'zope.browserpage',
+  'zope.browserresource', 'zope.cachedescriptors', 'zope.component',
+  'zope.configuration', 'zope.container', 'zope.contentprovider',
+  'zope.contenttype', 'zope.datetime', 'zope.deferredimport',
+  'zope.deprecation', 'zope.dottedname', 'zope.event', 'zope.exceptions',
+  'zope.filerepresentation', 'zope.globalrequest', 'zope.hookable',
+  'zope.i18n', 'zope.i18nmessageid', 'zope.interface', 'zope.lifecycleevent',
+  'zope.location', 'zope.pagetemplate', 'zope.processlifetime', 'zope.proxy',
+  'zope.ptresource', 'zope.publisher', 'zope.schema', 'zope.security',
+  'zope.sequencesort', 'zope.site', 'zope.size', 'zope.structuredtext',
+  'zope.tal', 'zope.tales', 'zope.testbrowser', 'zope.testing',
+  'zope.traversing', 'zope.viewlet'
+]
+'Products.CMFCore' = [
+  'docutils', 'five.localsitemanager', 'Missing', 'Products.BTreeFolder2',
+  'Products.GenericSetup', 'Products.MailHost', 'Products.PythonScripts',
+  'Products.StandardCacheManagers', 'Products.ZCatalog', 'Record',
+  'zope.sendmail', 'Zope'
 ]
 'plone.base' = [
-  'AccessControl', 'Products.BTreeFolder2', 'Products.CMFCore',
-  'Products.CMFDynamicViewFTI', 'zope.deprecation',
+  'plone.batching', 'plone.registry', 'plone.schema','plone.z3cform',
+  'Products.CMFCore', 'Products.CMFDynamicViewFTI',
```

Comment:
The idea is, if a package depends on plone.base it does not need to depend on CMFCore any longer. So that is why CMFCore is in plone.base. but it seems `z3c.dependencychecker` does not expand the user mappings, what I outlined here https://github.com/reinout/z3c.dependencychecker/issues/111

Reply:
But then, aside from my personal views, if we have to wait for https://github.com/reinout/z3c.dependencychecker/issues/111 then we should not merge this one right?

Reply:
> we should get regressions whenever Zope drops a dependency

In fact we could automate the creation of the list of dependencies for the three packages. 

Zope:
- fresh venv
- pip install Zope
- pip freeze to a file and reformat the output for TOML
- 
CMFCore:
- now pip install Products.CMFCore in the above venv
- pip freeze, diff with above output and take only new entries and reformat output for TOM

plone.base
- same procedure based on CMFCore venv

Reply:
> if we have to wait for [reinout/z3c.dependencychecker#111](https://github.com/reinout/z3c.dependencychecker/issues/111) then we should not merge this one right?

We can merge already, but the outcome wont be perfect. It is already much better than what we have already, so I would merge, even if we end up with more dependencies than needed.

Reply:
I like the idea of automating the dependencies generation, we can run it once a week on a GHA and if the dependencies are different than on what we have on `config/default/pyprojec.toml` warn about it ✨

Reply:
Let's merge then 👍🏾 we still have Jenkins and GHA that will alert is if something goes wrong anyway 🍀